osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/DSUpdateOrDeleteMetaTaskTestData.java

```diff
@@ -225,10 +225,12 @@ public static TaskGraph createDAIHostSelectedGraph(DeploymentSpec ds) {
 
     public static TaskGraph createDAIHostNotSelectedGraph(DeploymentSpec ds) {
```

Comment:
can you add a unit test for this specific scenario? Basically have multiple  DAI's and make sure the tasks are created appropriately.